raresGabriel01/home-assignment
# [ISSUE-10] Prettier pre-commit hook

# Issue link
#10 
# Description
Basic setup for a prettier pre-commit hook.
I also ran `npx prettier --write .`, so there will be multiple files modified in this PR.
# How to test?

1. Run some messy code
2. Checkout to a new branch
3. commit changes
4. Check if the messy code got formatted

# Attachments
N/A
